extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ServerLimitsConfig.java

```diff
@@ -25,4 +26,12 @@
      */
     @ConfigItem
     public Optional<MemorySize> maxChunkSize;
+
+    /**
+     * If present will configure the max length of the initial line (e.g. {@code "GET / HTTP/1.0"}).
+     * Otherwise will use Netty's default which is 4096.
+     */
+    @ConfigItem
```

Comment:
```suggestion
    @ConfigItem(defaultValue = "4096")
```

Let's add the default value so that it is visible in the generated doc.

Reply:
@machi1990 nice catch! I agree

Reply:
Does it make sense to make to use an OptionalInt if there's a default value?

Reply:
Nope, you can change to `int` in this case

Reply:
ok

Reply:
Done